template/en/default/account/prefs/mfa.html.tmpl

```diff
@@ -16,10 +23,17 @@
   [% RETURN %]
 [% END %]
 [% IF user.cryptpassword == '*' %]
-  <input type="hidden" name="mfa_action" id="mfa-action" value="">
+  <input type="hidden" name="mfa_action" id="mfa-action" value="" data-nopassword="true">
+  <p>
+    Your account requires MFA to be configured before you can use [% terms.Bugzilla %]
```

Comment:
Your account requires Multi-factor Authentication (MFA) to be configured before you can use [% terms.Bugzilla %].

Reply:
MFA or 2FA here?